scripts/main.js

```diff
@@ -210,6 +206,12 @@ function newImageUpload(uid, {fnName = "", active = true} = {}) {
     shadeMap: undefined
   }
 
+  $('[data-toggle="tooltip"]').tooltip();
```

Comment:
It [initializes](https://getbootstrap.com/docs/4.0/components/tooltips/#example-enable-tooltips-everywhere) the bootstrap tooltips (E.g. the help text on a questionmark symbol for the dither option and the new save button).
Since these elements are newly added in the image form, they need to be initialized too in the same call. The selector could probably be made more specific to this image form instead of for all tooltips on the page, I'll edit this in the next commit.